index.html

```diff
@@ -712,101 +712,79 @@ <h2 class='informative'>Vary</h2>
       </section>
     </section>
 
-    <section id="messaging">
-      <h2>Messaging</h2>
+    <section id="notifications">
+      <h2>Notifications</h2>
       <section class="informative">
         <h3>Introduction</h3>
         <p>
-          This section defines when messages are emitted by a Fedora
-          implementation, the minimal set of data contained in those messages and how
-          those data are serialized. These messages may occur synchronously or asynchronously
-          with the API operations that cause them to be emitted. They are typically used to
-          support external integrations. The structure of these messages draws upon the
+          This section defines when notifications are made available by a Fedora
+          implementation, the minimal set of data contained in these notifications and how
+          the data are serialized. Notifications may be emitted synchronously or asynchronously
+          with the API operations that cause them to be emitted. These notifications are typically
+          used to support external integrations. The structure of these notifications draws upon the
           existing [[activitystreams-core]] and [[ldn]] specifications.
-          An implementation is free to choose from any messaging technology so long as
-          the messages conform to what is described in the following sections.
-        </p>
-      </section>
-
-      <section id="emitting-messages">
-        <h3>Emitting Messages</h3>
-        <p>
-          Any operation that causes the state of a resource to durably change MUST cause a
-          message to be emitted with a corresponding resource identifier.
-        </p>
-        <p>
-          A failed operation (HTTP 4xx or 5xx) MUST NOT emit any messages.
-        </p>
-        <p>
-          Messages MUST NOT be emitted until after any changes have been persisted to durable storage.
+          An implementation is free to choose from any transport technology so long as
+          the notifications conform to what is described in the following sections.
         </p>
         <p>
-          Repository start, stop and configuration change operations MAY cause a message to be emitted.
+          Implementers should be aware that some operations cause multiple resources to change.
+          In these cases, there will be a corresponding notification describing each of the changes.
```

Comment:
@barmintor that is a good summary. Here is an explanation for each removal:

> "Repository start, stop and configuration change operations MAY cause a noun to be emitted."

The new (and old) language _requires_ notifications for resource modifications. As long as we don't explicitly forbid other events (which we really shouldn't) then allowing events for start/stop operations is implied and doesn't need to be called out explicitly.

> "nouns MAY contain a timestamp marking the time the resource was added/modified/deleted."

Activity Streams is flexible about the content. It is also true that messages MAY contain any other data, so it is superfluous to state that they MAY contain timestamps.

> "A failed operation (HTTP 4xx or 5xx) MUST NOT emit any nouns."

As per the discussion elsewhere, I don't think we should explicitly _restrict_ certain potential categories of notifications, especially since there are (arguably) use cases for them (e.g. audit)

> "nouns MUST NOT be emitted until after any changes have been persisted to durable storage."

This is removed because it implies a certain type of persistence layer for an implementation, which is inappropriate.

> Emitted events MUST contain exactly one value (cardinality removed) for resource identifier/IRI

Activity Streams allows for a Collection structure (multiple events in a single notification). Any AS/2.0 client can transparently handle these. If we explicitly require one-resource-identifier-per-event, we are implicitly forbidding the use of AS Collections, which would be a terrible thing.

> Emitted events MUST contain exactly one value for event identifier

This is an overreach of the Fedora spec. Transport mechanisms provide their own identifier and in any case a unique identifier may be added by any implementation. The only reason this was ever part of the spec is because the current Fedora4 implementation includes a unique identifier for messages (some sort of UUID). Note -- this is different than the resource identifier, which _is_ required.